src/app/modules/shared_models/emoji_reactions_item.nim

```diff
@@ -0,0 +1,22 @@
+import NimQml
+
+type
+    EmojiReactionItem* = ref object
+        emojiId*: int
+        filename*: string
+        didIReactWithThisEmoji*: bool
+
+proc initItem*(emojiId: int, filename: string, didIReactWithThisEmoji: bool): EmojiReactionItem =
+    result = EmojiReactionItem()
+    result.emojiId = emojiId
+    result.filename = filename
+    result.didIReactWithThisEmoji = didIReactWithThisEmoji
+
+proc emojiId*(self: EmojiReactionItem): int =
+    self.emojiId
+
+proc filename*(self: EmojiReactionItem): string =
+    self.filename
+
+proc didIReactWithThisEmoji*(self: EmojiReactionItem): bool =
+    self.didIReactWithThisEmoji
```

Comment:
since all of these fields are already public (have `*`), there's no need to define getters for them.

Reply:
I would rather leave make them non-public after second thought do provide some encapsulation for the emoji_reactions_items module.